qfieldsync/ui/map_layer_config_widget.ui

```diff
@@ -87,9 +87,57 @@
    <item>
     <widget class="QGroupBox" name="attachmentsRelationsGroupBox">
      <property name="title">
-      <string>Attachments and Relationships Settings</string>
+      <string>Feature Form, Attachments and Relationships Settings</string>
      </property>
      <layout class="QFormLayout" name="attachmentsRelationsLayout">
+      <item row="0" column="0" colspan="2">
+       <layout class="QHBoxLayout" name="valueMapButtonInterfaceLayout">
+        <item>
+         <widget class="QLabel" name="valueMapButtonInterfaceLabel">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+            <horstretch>1</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="text">
+           <string>Item threshold under which value map editor widgets will use a toggle buttons interface</string>
```

Comment:
```suggestion
           <string>Item threshold under which value map editor widgets will use toggle buttons interface</string>
```

I guess the "a" is incompatible with the plural "buttons"?

Reply:
I think a works here as it is a[n] interface.

Reply:
Ah, my poor English. Good!